spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/FieldValueCounterController.java

```diff
@@ -1,140 +1,140 @@
-/*

```

Comment:
why is this entire class showing as if being replaced in the diff? (formatting?)